internal/ngrokapi/clientset.go

```diff
@@ -45,7 +44,7 @@ type DefaultClientset struct {
 
 // NewClientSet creates a new ClientSet from an ngrok client config.
 func NewClientSet(config *ngrok.ClientConfig) *DefaultClientset {
-	v6Config := &v6.ClientConfig{
+	v6Config := &ngrok.ClientConfig{
```

Comment:
**Non-blocking**: I think this `v6config` can be removed and we just pass `config` to the `kubernetesOperatorClient`. We had `v6config` in here while we were supporting both `v5` and `v6`, but now with the awesome work you did to remove v5, we can remove it.